.github/workflows/tests.yaml

```diff
@@ -87,7 +97,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['ubuntu-latest', 'macos-latest']
+        os: ['ubuntu-latest']
```

Comment:
Was losing maxos-latest deliberate?

Reply:
Yes! This is in the job that runs on Python 2.7. It used to work but since we use the latest versions of the platforms then they gradually remove support of Python 2.7. I don't remember exactly what failed in this case, I think it wasn't going very far, not even reaching the package installs.

Reply:
Ok, thanks!